### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Accessibleenvironment_DesiredStorageExplorertheme_AI4D: Color contrast ratio is less than 4.5:1 for the place holder present on the search box after applying High Contrast black theme.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1986]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.4.3; #AI4D; #Win32;

Environment Details:
Desktop App: 
Windows Version: Windows10
NVDA Version: 2019.2
Windows10

**Steps to Reproduce:**
1. Launch the Azure Storage Explorer application 
2. Navigate to Search for resources search box.​
3. Open AI4D and check whether the color contrast ration is less than 4.5:1.

**Actual:**
 Color contrast ratio is less than 4.5:1 for the place holder present on the search box after applying High Contrast black theme.

**Expected:** 
 Color contrast ratio should be more than 4.5:1 for the place holder present on the search box after applying High Contrast black theme.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:** 
Low vision user unable to see the text while applyong any dark theme.

**MAS Reference:**
MAS 1.4.3:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
**Attachment for Reference**
![1986_A11y_AzureToolsStorageExplorer_Accessibleenvironment_DesiredStorageExplorertheme_AI4D](https://user-images.githubusercontent.com/53940787/65873579-0edb3e80-e3a1-11e9-9897-95b57cc4a0c4.png)

**Comment by craxal** [comment]
@nirmalshoo Your description indicates that you switched to "High Contrast - Black", but your screenshot only depicts "Dark". The color contrast is passing just fine on my end for "High Contrast - Black".

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified the issue on version 1.12.0 and the issue is no longer reproduced,hence close this bug.Please refer the attachment below
![image](https://user-images.githubusercontent.com/53940787/73736091-3f275500-4766-11ea-8e2f-f08a42d359a0.png)

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
